GOVERNANCE-elections.md

```diff
@@ -0,0 +1,274 @@
+
+## Charter
+
+The following responsibilities and powers belong to the Steering Committee.
+
+Technical governance is expected to be performed by the Maintainers of the
+various project areas and subprojects, as defined in the appropriate OWNERs files.
+
+* Delegate ownership of, responsibility for and authority over areas of the
+  project to specific entities.
+* Define, evolve, and defend the vision, mission and the values of the project.
+* Charter and refine policy for defining new community groups, and establish
+  transparency and accountability policies for such groups
+* Define and evolve project and group governance structures and policies.
+* Act as a final escalation point for disputes in any project repository or
+  group.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
+* Coordinate with the CNCF regarding usage of the PROJECTNAME brand and deciding
+  project scope, core requirements, and conformance, as well as how that brand
+  can be used in relation to other efforts or vendors.
+* Decide, for the purpose of elections, who is a member of standing, and what
+  privileges that entails.
+* Control and delegate access to and establish processes regarding all project
+  repositories, resources, and assets.
+<!-- assumes a separate Code of Conduct Committee, which is expected for
+ any project mature enough to need a Steering Committee-->
+* Appointing the members of the Code of Conduct Committee in order to ensure
+  responsiveness, diversity, and good judgement.
+
+## Delegated authority
+
+The Steering Committee may choose to delegate its authority to other committees
+as-needed. The committee currently recognizes this delegated authority for:
+
+- [TODO: List of Delegated Authorities]
+
+<!-- Above put a list of powers of the SC that are being handled by some other
+group or person.  For example, you may have a Marketing Committee that handles
+all work with CNCF marketing, or a different body that handles escalated
+technical disputes.-->
+
+## Committee Meetings
+
+The Steering Committee meets [TODO: frequency of meetings], or as-needed.
+Meetings are held online, and are public by default.
+
+Given the private nature of some of these discussions (e.g. privacy, private
+emails to the committee, code of conduct violations, escalations, disputes
+between members, security reports, etc.) some meetings are held in private.
+<!-- consider setting up a cadence of private vs. public meetings here-->
+
+Meeting notes are available to members of the
+[developer mailing list](TODO).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance or community procedures are posted as
+[issues in the community repo](TODO), and
+the Steering Committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their
```

Comment:
add a `There are [TODO: add a number here] Seats on the Steering Committee. These seats are held by an individual,`

Reply:
whoops drop this, looks like you have it covered later in the doc.